src/controllers/words.js

```diff
@@ -64,8 +69,21 @@ export const getWords = async (req, res, next) => {
       dialects,
       examples,
     };
-    let query = !strict ? searchIgboTextSearch(searchWord, isUsingMainKey) : strictSearchIgboQuery(searchWord);
-    const words = await searchWordUsingIgbo({ query, ...searchQueries });
+    let query = !strict
+      ? searchIgboTextSearch(
+        searchWord
+          .split('')
+          .filter((c) => c !== '"' && c !== "'" && !(searchWord.match(/'.*'/) || searchWord.match(/".*"/)))
+          .join(''),
```

Comment:
These lines that are returning a new value that's derived from `searchWord` can be moved out to another function to reduce repetitiveness.